Gemfile

```diff
@@ -46,10 +46,13 @@ group :development do
   gem 'web-console', '>= 3.3.0'
   gem 'listen', '~> 3.2'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem "better_errors"
```

Comment:
what are those?

Reply:
It changes the default rails errors to a better interface. It makes it easier to debug